Modules/Core/Common/include/itkGaussianDerivativeOperator.hxx

```diff
@@ -41,15 +41,11 @@ GaussianDerivativeOperator<TPixel, VDimension, TAllocator>::GenerateCoefficients
 
 
   // Calculate scale-space normalization factor for derivatives
-  double norm;
+  double norm{ 1.0 };
   if (m_NormalizeAcrossScale && m_Order)
   {
     norm = std::pow(m_Variance, m_Order / 2.0);
   }
-  else
-  {
-    norm = 1.0;
-  }
```

Comment:
Maybe just one line of code:

    double norm{ (m_NormalizeAcrossScale && m_Order) ? std::pow(m_Variance, m_Order / 2.0) : 1.0 };